blueoil/converter/frontend/base.py

```diff
@@ -28,7 +28,7 @@ def read(self, path: str) -> Graph:
 
         Args:
             path (str): Path to the file to be read
-        
+
```

Comment:
Are these whitespace diffs related to the move? If not I recommend not to include them.

Reply:
Thx, but please just ignore white spaces in the diff by the review URL `&w=1`. It's too annoying to avoid that manually.